### PR TITLE
feat: add price impact to sqs

### PR DESCRIPTION
**Pull request by p0mvn**
## What is the purpose of the change

Adds price impact to SQS.

To be merged after this PR is complete SQS side: https://github.com/osmosis-labs/osmosis/pull/7107

### ClickUp Task

[ClickUp Task URL](PASTE_CLICKUP_TASK_URL_HERE)

### CODE REVIEW

**Review by jonator — APPROVED** [review]

**Review by MaxMillington — APPROVED** [review]

**Comment by jonator** [comment]
Let's not merge until service is updated

**Comment by p0mvn** [comment]
Deployments are updated. Both stage and prod.

Once this PR is merged, users should start seeing it.

This PR is pending @osmosis-labs/qa ACK

**Pull request merged**
